--- a/src/xnu-4570.71.2/libkern/c++/OSKext.cpp	2018-10-18 23:33:37.000000000 -0400
+++ b/src/xnu-4570.71.2/libkern/c++/OSKext.cpp	2018-10-17 17:46:46.000000000 -0400
@@ -10116,6 +10116,21 @@
     return(result);
 }
 
+//Apple didn't release this function in public release. We add it to fix the prelinkedkernel issue. https://twitter.com/panicaII/status/1049906905576087552
+OSSymbol* IOSKCopyKextIdentifierWithAddress(vm_address_t address);
+OSSymbol* IOSKCopyKextIdentifierWithAddress(vm_address_t address) {
+    OSSymbol* sym = NULL;
+    OSKext* kext = OSKext::lookupKextWithAddress(address);
+    if (kext) {
+        sym = (OSSymbol*)kext->getIdentifier();
+        if (sym) {
+            sym->retain();
+        }
+        kext->release();
+    }
+    return sym;
+}
+
 #if PRAGMA_MARK
 #pragma mark Personalities (IOKit Drivers)
 #endif
